.github/issue_template/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Asking a question / requesting help
+about: Ask the community for help or advise
+title: '[Question] '
+labels: 'Question'
+assignees: ''
+
+---
+
+**Is your question related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**What specific question do you have?**
+A simple, one sentence question.
+
+**Describe solutions you've tried**
+A clear and concise description of what you have tried on your own already before asking others to spend their valuable time on answering your question.
```

Comment:
> before asking others to spend their valuable time on answering your question

This may come across as a bit aggressive, perhaps remove?

Reply:
done in https://github.com/fNIRS/snirf/pull/138/commits/6d16f1caa2ed5186c5f5597266c1f08fa3e9997c